### DISCUSSION TITLE
Disable background parallax during gameplay

### DISCUSSION THREAD

**Discussion by memesnightmares** [ppy/osu#16311] [Ideas]
This is really subtle, I've been playing lazer for a year and never noticed before it was pointed out to me today, but during gameplay the background shifts as you move your cursor. This seems very disorienting and like a feature that doesn't belong. Menu parallax is great, but in-game I think it hurts more than it helps.

**Comment by peppy** [comment]
I know this has come up in the past.

I'm not strictly against this, but I think some people may like it being how it is, or potentially increasing the strength. Will wait for more opinions.

**Reply by memesnightmares** [reply]
I will say that I've played lazer with bg dim at 80% for a year at this point and never even noticed, but I heard it mentioned in [a twitch stream](https://www.twitch.tv/videos/1251863393?t=1h2m26s) (1:02:26-1:04:30, unrelated feedback on lazer afterwards) and the general impression seemed to be rather negative, but I'm unsure how much value you put in feedback from a twitch steamer and their chat.

I do feel that off-by-default would be the much more sensible default, because it seems like it would be disorienting for most people, but of course it's better to base decision making off of a wide consensus than just me :P

**Reply by TheBrunoRM** [reply]
Why not just make it a setting? Strength could also be added as a setting. (by "a setting" I mean adding it to the settings menu)

**Reply by bdach** [reply]
"why not just make it a setting" is not acceptable. we cannot be adding settings for every little detail in the game.
